### ISSUE TITLE
Improvement: setting for => not selling if ROI reached and buy signal is still on

### ISSUE THREAD

**Issue by seansan** [freqtrade/freqtrade#318]
## Step 1: Have you search for this issue before posting it?

Not a bug

## Step 2: Describe your environment

  * Branch: Develop

## Step 3: Describe the problem:

Logic to add. Now if ROI is reached then BOT sells, but hey! maybe there is still a positie BUY signal
The result = buy, sell, buy, sell

Better would be to check the buy signal before reaching ROI checking

If BUY = still true, then keep it

Ofcourse we need to make this a setting: maybe people still want to SELL at ROI, but ont the other hand the bot will then auto back-buy because it also sees a BUY signal

### In short request

- create setting: ignore_roi_if_buy_signal_true = true
- if true: check buy signal first, if still positive for 1 trade, then continue (do not check ROI)
- if false: do not check buy signal first, check ROI, normal flow

**Comment by glonlas** [comment]
Thank you for this well-written feature description.

**Comment by xmatthias** [comment]
Closing this as it's included in #942
